functions/Get-DbaDbCheckConstraint.ps1

```diff
@@ -0,0 +1,124 @@
+function Get-DbaDbCheckConstraint {
+    <#
+        .SYNOPSIS
+            Gets database Check constraints
+
+        .DESCRIPTION
+            Gets database Checks constraints
+
+        .PARAMETER SqlInstance
+            The target SQL Server instance(s)
+
+        .PARAMETER SqlCredential
+            Allows you to login to SQL Server using alternative credentials
+
+        .PARAMETER Database
+            To get Checks from specific database(s)
+
+        .PARAMETER ExcludeDatabase
+            The database(s) to exclude - this list is auto populated from the server
+
+        .PARAMETER ExcludeSystemTable
+            This switch removes all system objects from the table collection
+
+        .PARAMETER EnableException
+            By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+            This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+            Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+        .NOTES
+            Tags: Databases
+            Author: Cláudio Silva ( @ClaudioESSilva | https://claudioessilva.eu)
+
+            Website: https://dbatools.io
+            Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
+            License: MIT https://opensource.org/licenses/MIT
+
+        .EXAMPLE
+            Get-DbaDbCheckConstraint -SqlInstance sql2016
+
+            Gets all database check constraints
+
+        .EXAMPLE
+            Get-DbaDbCheckConstraint -SqlInstance Server1 -Database db1
+
+            Gets the check constraints for the db1 database
+
+        .EXAMPLE
+            Get-DbaDbCheckConstraint -SqlInstance Server1 -ExcludeDatabase db1
+
+            Gets the check constraints for all databases except db1
+
+        .EXAMPLE
+            Get-DbaDbCheckConstraint -SqlInstance Server1 -ExcludeSystemTable
+
+            Gets the check constraints for all databases that are not system objects
+
+        .EXAMPLE
+            'Sql1','Sql2/sqlexpress' | Get-DbaDbCheckConstraint
+
+            Gets the check constraints for the databases on Sql1 and Sql2/sqlexpress
+    #>
+    [CmdletBinding()]
+    param (
+        [parameter(Mandatory, ValueFromPipeline)]
+        [Alias("ServerInstance", "SqlServer")]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [PSCredential]$SqlCredential,
+        [object[]]$Database,
+        [object[]]$ExcludeDatabase,
+        [switch]$ExcludeSystemTable,
+        [Alias('Silent')]
```

Comment:
This is not required in new commands...only there for backward support in current commands.

Reply:
Thanks @wsmelton .
Removed.